src/screens/ConfirmTransactionScreen.js

```diff
@@ -13,25 +13,28 @@ const ConfirmTransactionScreen = ({ navigation }) => {
   const [processing, setProcessing] = useState(false);
 
   const submitTransaction = async (amount, toAddress) => {
-    setProcessing(true)
+    setProcessing(true);
```

Comment:
should we call dispatch loading here?

Reply:
Agreed. This should be stored in local state for now.